### PR TITLE
Suggested change

### PR DESCRIPTION
**Pull request by badersur**
Maybe first paragraph could be changed to:

.NET Core runs on Windows, Linux, and macOS / OS X. Install it on your machine and follow any of the following tutorials.

### CODE REVIEW

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/core/getting-started?branch=pr-en-us-1052](https://review.docs.microsoft.com/en-us/dotnet/articles/core/getting-started?branch=pr-en-us-1052)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C20%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609202117134984-1052%5Cworkflow_report.html).

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/core/getting-started?branch=pr-en-us-1052](https://review.docs.microsoft.com/en-us/dotnet/articles/core/getting-started?branch=pr-en-us-1052)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C20%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609202230379844-1052%5Cworkflow_report.html).

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/core/getting-started?branch=pr-en-us-1052](https://review.docs.microsoft.com/en-us/dotnet/articles/core/getting-started?branch=pr-en-us-1052)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C20%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609202233464581-1052%5Cworkflow_report.html).

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/core/getting-started?branch=pr-en-us-1052](https://review.docs.microsoft.com/en-us/dotnet/articles/core/getting-started?branch=pr-en-us-1052)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C20%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609202238032007-1052%5Cworkflow_report.html).

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/core/getting-started?branch=pr-en-us-1052](https://review.docs.microsoft.com/en-us/dotnet/articles/core/getting-started?branch=pr-en-us-1052)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C21%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609210148432597-1052%5Cworkflow_report.html).

File: docs/core/getting-started.md [hunk 1]
```diff
@@ -14,45 +14,23 @@ ms.assetid: d1453a0d-317c-4da6-b68e-422a2c0bfe49
...
-* [Getting started with .NET Core using the command line](tutorials/using-with-xplat-cli.md) - Use any code editor with the [.NET Core cross-platform command-line interface (CLI)](tools/index.md).
-
-See the [Prerequisites for Windows development](windows-prerequisites.md) topic for a list of the supported Windows versions. 
-
-## Linux
-
-Install Linux on your distribution/version:
+.NET Core runs on [Windows](windows-prerequisites.md), Linux, and macOS / OS X. Install it on your operating system using the following links:
 
+* [Windows 7 - 10 & Server 2008 - 2016](https://www.microsoft.com/net/core#windows)
```
Comment (badersur) [inline docs/core/getting-started.md]: Maybe change it to:

* [Windows 7 SP1, 8.1, 10 & Windows Server 2008 - 2016](https://www.microsoft.com/net/core#windows) (Read the [Prerequisites for Windows development](windows-prerequisites.md) topic for more details.)

    > OR: Read the [Prerequisites for Windows development](windows-prerequisites.md) topic for more details.

**Comment by badersur** [comment]
Because Core [getting started](https://docs.microsoft.com/en-us/dotnet/articles/core/getting-started) is just linking to how to install .NET Core and the [tutorials](https://docs.microsoft.com/en-us/dotnet/articles/core/tutorials/), then what do you think about:

1. updating Tutorial section to link to https://www.microsoft.com/net/core like:
    > [Install .NET Core on your platform of choice](https://www.microsoft.com/net/core).

    > Windows users: Read the [Prerequisites for Windows development](https://docs.microsoft.com/en-us/dotnet/articles/core/windows-prerequisites) for more details.

    > .NET Core isn't yet supported by macOS Sierra, but will be soon.

2. remove getting started and link to tutorials everywhere.

If these are not appropriate then I think we should include a section about [Docker](https://www.microsoft.com/net/core#docker) as well.

cc: @mairaw

**Comment by richlander** [comment]
The main effect of this change is removing the OS-specific headings. I think that the headings are useful rather than merging everything together.

Feel free to make other changes to this doc, however, I want to retain the headings. Closing the PR.

**Pull request closed without merging**